cli/commands/run/version_check.go

```diff
@@ -27,6 +31,8 @@ var TerraformVersionRegex = regexp.MustCompile(`^(\S+)\s(v?\d+\.\d+\.\d+)`)
 
 const versionParts = 3
 
+var versionFiles = []string{".terraform-version", ".tool-versions", "mise.toml", ".mise.toml"}
```

Comment:
Do you think we should make a ticket to backlog making this configurable? Maybe a user can just feed Terragrunt the files that might be used to control TF versions outside these defaults?

Reply:
Will raise another ticket for this